### ISSUE TITLE
re-add bcm2710-rpi-cm3.dtb in device tree for raspberrypi3-64, since dts now exists

### ISSUE THREAD

**Issue by stevep-korg** [agherzan/meta-raspberrypi#304]
The device tree entry for bcm2710-rpi-cm3.dtb should be re-added to raspberrypi3-64.conf when possible, because the companion dts file was recently made available in the raspberrypi linux repo, in this change:

https://github.com/raspberrypi/linux/commit/248e0f711ec517aff69b815bac7b884bd3a04679

So, once meta-raspberrypi layer is using code at this change or newer, this fix could be applied.  At that time, I could test that it builds, generate a pull request and it could be merged.

The current problem is that raspberrypi3-64 has no bcm2710-rpi-cm3.dtb entry in the kernel device tree, because there was previously no dts available to build it.

The cm3 should be supported because it is the industrial/commercial version of the R Pi 3B.

**Comment by stevep-korg** [comment]
This issue is related to issue 97:

https://github.com/agherzan/meta-raspberrypi/issues/97

**Comment by stevep-korg** [comment]
Closing this bug, because this fix would be problematic for users who are trying to build for a Linux 4.9 version.  In that case the cm3 dts is not available in that linux /raspberry pi repo branch, and that branch is seldom updated.

Instead, it's easy enough to add the cm3 dtb to the RPI_KERNEL_DEVICETREE_append variable in local.conf.  I have tested this and it works fine in sumo at the change Id indicating version 4.14.68.
